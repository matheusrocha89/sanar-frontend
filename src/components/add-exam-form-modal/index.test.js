import React from 'react';
import renderer from 'react-test-renderer';

import AddExamFormModal from './';


describe('AddExamFormModal', () => {
  it('renders correctly', () => {
    const show = false;
    const handleClose = jest.fn();
    const addNewExam = jest.fn();
    const tree = renderer
      .create(<AddExamFormModal show={show} handleClose={handleClose} addNewExam={addNewExam} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
