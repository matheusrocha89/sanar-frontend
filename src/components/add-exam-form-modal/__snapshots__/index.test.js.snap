// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddExamFormModal renders correctly 1`] = `null`;
